token.go

```diff
@@ -4,9 +4,30 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"strings"
+	"sync/atomic"
 	"time"
 )
 
+const (
+	DisablePadding uint64 = 0 // Utilizes RawURLEncoding
+	AllowPadding   uint64 = 1 // Utilizes URLEncoding
+
+)
+
+var decodePaddingAllowed uint64
+
+// SetDecodePadding will switch the codec used for encoding/decoding JWTs respectively. Note that the JWS RFC7515
+// states that the tokens will utilize a Base64url encoding with no padding. Unfortuneately, some implementations
+// of JWT are producing non-standard tokens, and thus require support for decoding. Note that this is a global
+// variable, and updating it will change the behavior on a package level.
+func SetDecodePadding(setPadding uint64) {
+	atomic.SwapUint64(&decodePaddingAllowed, setPadding)
+}
+
+func init() {
+	SetDecodePadding(DisablePadding)
```

Comment:
Is that even necessary? it probably should be automatically init'ed as a 0 in the declaration above?

Reply:
Agreed, removing this!